src/asInput/index.jsx

```diff
@@ -204,6 +209,30 @@ const asInput = (WrappedComponent, inputType = undefined, labelFirst = true) =>
       );
     }
 
+    renderInputGroupAddons(type) {
+      // Type is either inputGroupPrepend or inputGroupAppend
+      const formattedType = type.replace('inputGroup', '').toLowerCase();
```

Comment:
This logic feels brittle - if we change the prop name to `inputGroupsAppend` for example, we have to remember to change this method.

Reply:
Yeah, I'm not a huge fan on this logic either. I was trying to have one method that can handle both `inputGroupPrepend` and `inputGroupAppend`. A better alternative might be to split it up into 2 methods for each type of input group addon.